### DISCUSSION TITLE
3.8.1

### DISCUSSION THREAD

**Discussion by Davidy22** [Guake/guake#1955] [Announcements]
3.8.1 is a bugfix release mainly targeting a major regression in keyboard shortcuts, though it also comes with some minor added features and updates. More bugfix updates are in the works to resolve other less pressing issues that didn't get addressed in this release.

In other news, an officially endorsed PPA is in the works for Guake as an upcoming alternative method to install Guake. Watch this space for updates.

### Release Summary

Added line height and character width adjustment options

Fix keybinding regressions

Lazy focus loss to avoid closing on short-term losses

Moved guake.desktop.metainfo.xml to \$datadir

### New Features

- - Feature request: Setting line-spacing / line-height #849

- - Support of short term focus loss and avoid folding #1643

### Bug Fixes

- - Some shortcut keys are abnormal #1939

### Notes for Package Maintainers

- Glade layout file upgraded with 3.38.2 from last used 3.22.2, should still be valid but some things may be deprecated/renamed.

- guake.desktop.metainfo.xml has been moved to \$datadir in Makefile

<hr /><em>This discussion was created from the release <a href='https://github.com/Guake/guake/releases/tag/3.8.1'>3.8.1</a>.</em>

**Comment by gsemet** [comment]
hi. if we look at the github actions, it looks like the job that should upload to pypi is skipped or not executed. it uploads well to the pypi-test, but it doesn't on the real pipy !

**Reply by Davidy22** [reply]
Yeah, I'm looking at the action and there's just nothing that looks like a pypi upload. Can take a look at CI sometime and see why

**Reply by mlouielu** [reply]
Isn't that intended as we comment pypi upload at here?

https://github.com/Guake/guake/actions/runs/1381671106/workflow#L71

**Reply by Davidy22** [reply]
Oh, so it is. I am a little fuzzy on the pip CI steps having not written them. Triggered by tagging it looks like, too late for it to run for 3.8.1, but I intend to run a new release fairly soon after some more bugfixes so we could be able to see if it works soon enough. Anyone against uncommenting and seeing how it goes next release? It does mean we may need to have quick hands on pypi, in case the script messes it up.

**Reply by mlouielu** [reply]
I think we can tag another release candidate for testing production-pypi script. (3.8.2-rc1).

I didn't catch up for 3.8.0 release, so 3.8.0 is a manual release to pypi?

**Reply by Davidy22** [reply]
But then we'd have a version labelled rc up on pypi, and for a release that'd presumably be a minor release

3.8 hasn't been released at all to pypi yet, but it would have had to be manual because the lines were commented out for that release too.

**Reply by gsemet** [reply]
I have   updated the release process and updated the translation in https://github.com/Guake/guake/pull/1956, but the pipeline fails for a reason i don't understand yet. Can you take over it? And once it is fixed, if you want to release, please use the following syntax: 3.8.2.0rc0, 3.8.2.0rc1, 3.8.2.0rc2,...

rc should be uploaded to pypi (the real one) on tag. once this works, we can put the real tag

**Reply by Davidy22** [reply]
Oh, would we be labelling rc1, rc2, etc to test that the pypi upload works? Yeah, sure. I'll take a look at the action as well in the meantime.

**Reply by gsemet** [reply]
this will only make 3.8.2.0rc0 unvisible through pip install (only if you really want prerelease)
